src/EvaluationDetails.php

```diff
@@ -87,11 +87,11 @@ public function getError(): ?string
     }
 
     /**
-     * @return int the last download time of the current config in unix seconds format.
+     * @return float the last download time of the current config in unix milliseconds format.
      */
-    public function getFetchTimeUnixSeconds(): int
+    public function getFetchTimeUnixMilliseconds(): float
```

Comment:
This is a BC to a public API, so I guess it should be mentioned in the release notes.